### DISCUSSION TITLE
Just a question: About how to configure latexmk with multiple glossaries

### DISCUSSION THREAD

**Discussion by grivasgervilla** [James-Yu/LaTeX-Workshop#3096] [Q&A]
Hi,

first of all, many thanks for this awesome extension, which I extensively use in my work. Nowadays, I using it to write my PhD thesis and I think that it is the best extension for LaTeX.

I have I doubt about how you have configured latexmk to work with multiple glossaries (using the `glossaries` LaTeX package). Since, when I compile my LaTeX project with the default latexmk recipe, I have no problem with glossaries.

However, recently I am using my own Makefile thanks to the `latex-orkshop.latex.external.build.command` option. In this Makefile, I use latexmk as well to compile my project. However, I have to invoke `makeglossaries` manually in order to get my glossaries generated and updated on the compilation process.

My question is how have you done this in your recipe? Which options of latexmk do you use in order to make it work properly with multiple glossaries? Could I see your code for this?

Many thanks for your time. :smile:

**Comment by tamuratak** [comment]
This issue is not related to the extension. We cannot help you. Please find help somewhere else.
